crash_similarity.py

```diff
@@ -86,31 +110,36 @@ def get_stack_trace_for_uuid(uuid):
 
 
 def train_model(corpus):
-    if os.path.exists('stack_traces_model.pickle'):
-        return gensim.models.Doc2Vec.load('stack_traces_model.pickle')
+    
+    if checkif_model_trained_in_last_24hours()==True:
+        print 'Model already trained in last 24 hours \n'
```

Comment:
you should use `print()` to make code compatible with python3.
why space and \n at the end?